homework.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
@lemonhawk21 When you open this file in the browser, you'll be able to see the `console.log()` output of your homework!